### ISSUE TITLE
Multi-domain particles in misc/pip is not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [adda-team/adda#156]
```
Currently the 'pip' program generates only single-domain particles, just 
ignoring the domain (material) information in the input file (e.g. .obj file). 
However, the underlying routine ivread_wr.f90 seems to support different 
materials in input files, so it should be quite easy to modify the main 'pip' 
program to properly account for different materials.
```

Original issue reported on code.google.com by `yurkin` on 12 Dec 2012 at 2:59

**Comment by stefaniagl** [comment]
[two_domain_particle.zip](https://github.com/adda-team/adda/files/7250825/two_domain_particle.zip)

**Comment by stefaniagl** [comment]
[two_domain_particle.zip](https://github.com/adda-team/adda/files/7252931/two_domain_particle.zip)

**Comment by myurkin** [comment]
Fixed by 94e03da
